bin/cron/start_broken_link_checker.rb

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+require_relative '../../deployment'
+
+def main
+  `pkill -9 -f brokenLinkChecker`
+  `#{deploy_dir('tools', 'scripts', 'brokenLinkChecker', 'brokenLinkChecker.js')}`
```

Comment:
Actually it seems likely that if the node process doesn't exit then now we'll just have a ruby process lingering as well.  The solution might be to detach the execution as described here: https://stackoverflow.com/a/10021735

Reply:
Actually `exec` will replace the ruby process with the node process, which seems a cleaner solution.  https://stackoverflow.com/a/18623297

Reply:
..though this probably means that output will not be captured by cron to be emailed back, so the `notify` entry can be removed.  We seem to rely on the Slack channel to observe progress these days, so that should be fine.